npm.css

```diff
@@ -0,0 +1,229 @@
+  width: 40px;
+  margin-right: 1rem;
+}
+
+searchBar {
+  width: 600px;
+}
+
+searchBar ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+searchBar a {
+  text-decoration: none;
+  color: black;
+}
+
+#form {
+  background:rgba(0,0,0,.05);
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+  border: none;
+}
+
+#submit {
+  background-color: #fb3e44;
+  color: #fff;
+  text-decoration: none;
+  cursor: pointer;
+  border:none;
+  padding: 8px 12px;
+  font-family: 'Poppins';
+}
+
+#submit:hover {
+  
+  background:pink;
+  box-shadow: 0px 0px 1px #777;
+}
+
+.logoSize {
+  width: 50px;
+  margin-right: 1rem;
+}
+
+.glassSize {
+  width: 25px;
+  margin-right: 1rem;
+  background:rgba(0,0,0,.05);
+}
+
+.searchBarButton {
+  color: #fff;
+  text-decoration: none;
+  cursor: pointer;
+  border:1px solid black;
+  padding: 6px 12px;
+  font-family: 'Poppins';
+}
+
+a.searchBarButton:link, a.searchBarButton:link{
+  color: #fff;
+  text-decoration: none;
+  cursor: pointer;
+  border:1px solid black;
+  padding: 6px 12px;
+  font-family: 'Poppins';
+}
+
+h1{
+  text-align: center;
+  font-size: 4em ;
+  font-weight: bold;
+}
+
+#firstSection {
+  display: flex;
+  flex-flow: row wrap;
+  flex-direction: column;
+  align-content: center;
+  background-color: rgba(232,217,217,.3)
+  width: 600px;
+}
+
+.linkHighlight {
+  width: 300px;
+  font-weight: bold;
+  color: white;
+  background-color: #FB3B49;
+  text-align: center; 
+  padding: 10px;
+  box-shadow: 8px 8px 0 rgba(251,59,73,.2);
+
+}
+
+.linkDontHighlight {
+  width: 300px;
+  font-weight: bold;
+  background-color: rgba(232,217,217,.3);
+  text-align: center;
+  padding: 10px;
+}
+
+.article {
+  width: 600px;
+  text-align: center;
+}
+
+#secondSection {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(232,217,217,.3);
+  /*width: 600px;
+  height: 600px;*/
+}
+
+h2{
+  text-align: center;
+  font-size: 2.5em ;
+  font-weight: bold;
+  font-style: italic;
+  background-color: rgba(255,204,53,.4);
+  transform: skew(9deg,0deg);
+  padding: 10px;
+}
+
+h3 {
+  color: #ED1C24;
+  font-weight: bold;
+  text-align: left;
+}
+
+#thirdFlex {
+  display: flex;
+  justify-content: space-around;
+  padding: 150px 50px;
+  height: auto;
+  /*width: 200px;*/
+  align-items: center;
+}
+
+.iconBox {
+  width: 200px;
+}
+
+.twoButtons{
+  display: flex;
+  justify-content: center;
+}
+
+.hexagon{
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 20%;
+  padding: 40px;
+}
+
+.createOrg{
+  width: 250px;
+  height: auto;
+  font-weight: bold;
+  color: white;
+  background-color: black;
+  text-align: center; 
+  padding: 10px ;
+  box-shadow: 8px 8px 0 rgba(128,83,35,.2);
+  font-family: 'Poppins';
+ }
+
+.end{
+  display: flex;
+  justify-content: center;
+  color: white;
+}
+
+a.end:link, a.end:visited {
```

Comment:
This is not selecting any html element, you can simply add a class to the element you want to style and use this class in CSS.